packages/@aws-cdk/aws-lambda/test/function.test.ts

```diff
@@ -3152,6 +3152,29 @@ test('set SnapStart to desired value', () => {
   });
 });
 
+test('enable SnapStart on L2 lambda construct', () => {
```

Comment:
```suggestion
test('function using SnapStart', () => {
```

Reply:
> Please add an integ test for this change.

Any suggestion how this can be done ? To generate the snapshot I need a java zip file for the lambda function. Do cloudFormation/lambda services currently have a validation to check the runtime for snapStart ?

Reply:
I have added integ tests for this change. Things should be better now ! Could you please review the PR whenever possible ?